CHANGELOG.md

```diff
@@ -4,3 +4,7 @@
 
 ### Added
 - Initial commit.
+- New Class `Endpoint` for wrapping generic endpoints.
+
+### Removed
```

Comment:
Since this will be the initial version to be released, do we need to mention what was removed?

Reply:
I'm adding the changelog more because we have made so many changes so it makes it easier to keep up with. Also, I think the initial release should be the ES6 one.

Reply:
I feel the same as you about having more entries in changelog. It's just that adding entries about what we have removed while we haven't even published the first version of the package, it seems odd. I think what we can do here is instead of mentioning what was removed, maybe under `Added` we should mention about `filterByTag` and some texts about its usage.

Reply:
Good idea and addressed here: https://github.com/w3c-ccg/vc-api-test-suite-implementations/pull/32/commits/efd2274f0046be48b802c7bc35c719c88a8046c8